src/diagnostics/logPointsSessionWizard.ts

```diff
@@ -1,331 +1,321 @@
-import { SubscriptionModels } from 'azure-arm-resource';

```

Comment:
why is this entire files showing as a diff?  Are the line-endings messed up?

Reply:
It should be right now.